examples/deployment/algob.config.js

```diff
@@ -67,6 +90,7 @@ let defaultCfg = {
 module.exports = {
   networks: {
     localhost: defaultCfg,
-    default: defaultCfg
+    default: defaultCfg,
+    KMD: KMDConfig
```

Comment:
this should be a part of a network configuration.

Reply:
ok